### DISCUSSION TITLE
x-logo-url an Example of Vendor Extensions

### DISCUSSION THREAD

**Discussion by BlackRhinoMedia** [DarkaOnLine/L5-Swagger#440] [General]
Could we add this as an example somewhere to make it easy for people to understand how to get a Logo into their documentation?

Sorry the formatting below isn't coming through properly the code [block can be seen here](https://stackoverflow.com/questions/70897715/l5-swagger-swagger-php-support-for-x-logo-within-the-openapi-3-0-1-in-the-oa)

`/**
 * @OA\Info(
 *   title="Example",
 *   version="1.0.0",
 *   x={
 *     "logo": {
 *       "url": "path_to_your_logo"
 *     }
 *   }
 * )
 */`

A user kindly pointed me in the right direction on Stack Overflow - I was searching for Logo but the documentation doesn't currently mention an example for it? I think it would help to go on [this page perhaps](https://zircote.github.io/swagger-php/Getting-started.html#vendor-extensions)?

**Comment by DarkaOnLine** [comment]
Sure we can. We could add this to our wiki, examples: https://github.com/DarkaOnLine/L5-Swagger/wiki/Examples
Or to the annotations - https://github.com/DarkaOnLine/L5-Swagger/blob/master/tests/storage/annotations/OpenApi/Anotations.php
